pandas/core/frame.py

```diff
@@ -8271,7 +8271,7 @@ def explode(
 
         columns: list[str | tuple]
         if is_scalar(column) or isinstance(column, tuple):
-            assert isinstance(column, (str, tuple))
+            assert isinstance(column, (str, tuple, int))
```

Comment:
Add tests first

We should check for scalar values

Reply:
actually i would remove the assert entirely i think (but agreed on tests), e.g. you can in theory have a datetime here as well

Reply:
Yes hence the scalar idea, but yeah removing is ok too